source/process/release-process.md

```diff
@@ -21,197 +21,179 @@ Pre-work for the current release begins at the code complete date of the previou
 
 ### B. (T-minus 15 working days) Cut-off for submitting major features
 
-No pull requests for major features should be **submitted** to the current release after this date (except if release manager decides to add "release-exception" label to Jira ticket).
+No pull requests for major features should be **submitted** to the current release after this date. Exceptions can be made by the Release Manager in special cases. 
 
 1. Release Manager:
-    - Post this checklist in Release channel
+    - Post this checklist in Release Checklist channel
 2. PM:
     - Prioritize reviewing major features, ensuring any bugs and UX issues get fixed
     - Check that all major features are behind a feature flag
+    - Confirm with Leads that each Enterprise feature is in the correct [pricing SKU](https://about.mattermost.com/pricing/), if not alert the release manager
 3. Dev:
     - Prioritize reviewing, updating, and merging of pull requests for major features
-4. Marketing:
-    - Confirm each Enterprise feature is in the correct [pricing SKU](https://about.mattermost.com/pricing/), if not alert the release manager
 
 ### C. (T-minus 12 working days) Cut-off for merging major features
 
-No pull requests for major features should be **merged** to the current release after this date (except if release manager decides to add "release-exception" label to Jira ticket).
+No pull requests for major features should be **merged** to the current release after this date. Exceptions can be made by the release manager in special cases.
 
-1. Release Manager:
-    - Post this checklist in Release channel
+1. **(Team) Major Feature Complete Meeting (10:00am San Francisco time)**:  
+    - Discuss worst bug currently on master
+    - Discuss release highlights for marketing
+    - Review status of last feature PRs to be merged
+2. Release Manager:
+    - Post this checklist in Release Checklist channel
     - Verify all items in the last posted release checklist are complete
-2. Logistics:
-    - Start posting a daily Zero Bug Balance query
-    - Queue an item for UX meeting to discuss worst UX bug and to do a 10-minute UI/UX bug bash
```

Comment:
Should we replace UX bug bash with something?

Reply:
@lindalumitchell perhaps there's a UI bug bash included in the release testing spreadsheet, for the UX team?

0/5

Reply:
I do have bug bashes included on some of the tabs, yes, but not specifically for the UX team and not on every tab. And/or worth having a bug bash at a platform meeting around this time in the process?

Reply:
I wasn't able to make a decision on this since I'm not sure how important UX bug bash has been in the past - if it is something we should definitely keep, we could do it in the platform meeting as suggested by LM?